cpp/ql/test/query-tests/Security/CWE/CWE-193/test.cpp

```diff
@@ -1,7 +1,7 @@
-char *malloc(int size);
+void* malloc(unsigned long size);
```

Comment:
Nit: Can we make the argument explicitly a `size_t` to handle 64-bit sizes?

Something like `using size_t = decltype(sizeof 0);` should do it, since I guess we don't want to include a header with the actual definition.

Reply:
Oh yeah we can definitly do that. Let me fix that in another PR right away